src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java

```diff
@@ -166,6 +171,12 @@ public void drawFG( int offsetX, int offsetY, int mouseX, int mouseY )
 	{
 		this.fontRendererObj.drawString( this.getGuiDisplayName( GuiText.CraftingStatus.getLocal() ), 8, 7, 4210752 );
 
+		if( this.craftingCpu.eta > 0 )
+		{
+			final String etaString = this.getGuiDisplayName( GuiText.ETA.getLocal() ) + " " + DurationFormatUtils.formatDuration( TimeUnit.MILLISECONDS.convert( this.craftingCpu.eta, TimeUnit.NANOSECONDS ), this.getGuiDisplayName( GuiText.ETAFormat.getLocal() ) );
+			this.fontRendererObj.drawString( etaString, 8, 165, 4210752 );
```

Comment:
Already considered it. If I would know what `4210752` actually means...
It is really something which should be done completely or not at all. Some half done state where some magic numbers are moved to constants with meaningful names and some not is actually worse in my opinion.